spellbook/tests/safe/eth_transfers_check_sum.sql

```diff
@@ -0,0 +1,11 @@
+-- Check that Eth transfers amount for safes on a specific date is correct
+
+with unit_tests as (
+    select round(sum(amount_raw) / 1e18, 0) as total
+    from {{ ref('safe_ethereum_eth_transfers') }}
+    where block_time between '2022-01-01' and '2022-01-03'
+)
+
+select case when total = 2996 then true else false end as test
+from unit_tests
+where test = false
```

Comment:
you can't refer back to 'test' in the same query here, you'll either need to reuse the case logic fully in the where clause or move this column up into the CTE above, then refer here

Reply:
Thanks! Changed